### DISCUSSION TITLE
Release 5.3.3

### DISCUSSION THREAD

**Discussion by miguel-a-calles-mba** [serverless-heaven/serverless-webpack#1549] [Team Posts]
All the PRs are closed for milestone 5.3.3.

https://github.com/serverless-heaven/serverless-webpack/milestone/31

Are there any other PRs we should add?

Should we publish `5.3.3-beta.0` or go straight to `5.3.3`?

Thoughts?

**Comment by miguel-a-calles-mba** [comment]
Just following up. Are we good to release 5.3.3?

**Comment by j0k3r** [comment]
Go ahead and release! :)

**Comment by miguel-a-calles-mba** [comment]
Release 5.3.3 just went out.
